device.mk

```diff
@@ -150,9 +150,24 @@ PRODUCT_PACKAGES += \
     android.hardware.health@1.0-impl \
     android.hardware.health@1.0-service
 
-# Keymaster HAL
+# Security
```

Comment:
Maybe it is a good point to move all security into security.mk
And guard all non generic dependencies(like OPTEE_PLATFORM_FLAVOR, OPTEE_PLATFORM and *.ta ??) with TARGET_BOARD_PLATFORM.

Reply:
Done.